### DISCUSSION TITLE
Please include Effects, Scrollbars, etc. in the Accessibility Settings

### DISCUSSION THREAD

**Discussion by MarjaE2** [linuxmint/discussions#31] [Ideas]
Hi,

I get migraines from a lot of common types of flashing and animation. I checked under Accessibility, and did not find an option to disable these. I eventually found they have another section in Cinnamon, which can disable most (but not enough) of these, but searching through different system settings for different accessibility fixes is awkward.

I also struggle with narrow scrollbars. I searched for scrollbars in the search section in the main Cinnamon window, but didn't see anything. I also quit Linux, booted into another system, and checked the pdf manual, and didn't see anything about scrollbars there either. I can't read readthedocs.io so I don't know if there's anything elsewhere in the documentation. It turns out they're under Themes, but it could be a bit easier for users to find them...?

I think the Accessibility Settings could include another way to disable Effects, and another copy of the scrollbar settings from Themes.

Note that disabling Effects does not keep Cinnamon from showing, hiding, or flashing the scrollbars. I'm not sure if there is an option to fix that.

**Comment by JosephMcc** [comment]
> Note that disabling Effects does not keep Cinnamon from showing, hiding, or flashing the scrollbars. I'm not sure if there is an option to fix that.

It's in the same place where you change the scrollbar size, under Themes. You can disable the overlay scrollbars there.

**Reply by MarjaE2** [reply]
I tried that, but I still encountered a lot of these. I don't know if it requires logging out and logging back in or something.

**Comment by mtwebster** [comment]
Disabling overlay scrollbars requires logging out/in, changing the scrollbar widget requires restarting the affected programs (or log out/in).

I've got some usability issues, too - unfortunately not every program respects these changes. Some (like firefox) offer their own way to disable overlay scrollbars (about:config: widget.gtk.overlay-scrollbars.enabled) but not change their width.

You can also increase the 'draggable' area around a window under Window settings -> Behavior.

I might consider moving all of this into accessibility also. I originally considered them more as convenience things, but they probably make more sense there.

**Comment by Secret-chest** [comment]
There should also be an effect "kill switch".
